### ISSUE TITLE
Establishment of localization & pathogenesis

### ISSUE THREAD

**Issue by PhiBabs935** [infectious-disease-ontology-extensions/ido-core#25]
Currently, 'establishment of localization' has the axiom: part of SOME pathogenesis.

This results in a false implication I believe. Not all 'establishment of localization in a host' and 'colonization of host' are pathogenic processes. As in the case of colonization by commensal microorganisms. 

This is an issue that needs to be sorted out. I also wonder whether the axiom 'preceded by' some 'pathogen transmission process' is really correct for 'establishment of localization in host'/'colonization of host'. It certainly should not be asserted for 'establishment of localization', as this process is far more general in scope than pathogens and microorganisms.

**Comment by PhiBabs935** [comment]
Related, currently in IDO the axiom {'part of' SOME 'pathogenesis'} is asserted for both 'pathogen transmission process' and 'process of establishing an infection'.

Isn't that incorrect? We are currently defining 'pathogenesis' in the ontologies as a process that has the process parts, 'pathogen transmission process', 'establishment of localization in host', 'process of establishing an infection' and 'appearance of disorder'. But if we assert that 'pathogen transmission process' and 'process of establishing an infection' are 'part of' pathogenesis, this implies that any pathogen transmission and subsequent infection is part of a process that leads to a disorder, which of course is not true (the case of HIV infection in the resistant host with a gene mutation).

Of course 'pathogenesis' can be connected to the above processes by 'has part', but we should not say the inverse.

**Comment by johnbeve** [comment]
My thinking was that, say, all pathogen transmission processes are part of some pathogenesis, and pathogenesis was meant to potentially have proper process part. The textual definition needs to be updated to reflect that. I didn't assert many axioms for pathogenesis, though I did say things like 'all viral replication has part some viral replication'

**Comment by PhiBabs935** [comment]
So, it only has the later parts potentially? But not necessarily? So to be clear, pathogenesis should NOT be defined so that it implies that it always involves appearance of disorder?

**Comment by johnbeve** [comment]
Apologies, I typed before checking my notes. Pathogenesis is a success term, so it does have at least those process parts. The parthood relation between proper parts should go the other way as you suggest, e.g. all pathogenesis has part some pathogen transmission process, etc. Good catch!

**Comment by PhiBabs935** [comment]
Yeah, I was just typing a reply about it being discussed as a success term in the VIDO paper, which I just re-checked!

Just a note for the future. This will require change to figure 6 in the VIDO paper (changing 'has part' to 'part of' and reversing the direction of the arrows).

**Comment by johnbeve** [comment]
Ha, same! I also made a note to change figure 6 :)

**Comment by PhiBabs935** [comment]
Cool, gonna make the changes in my OWL files.

**Comment by PhiBabs935** [comment]
Not so quick actually. What about bacteria causing disorder via toxin production without pathogen transmission? It seems like that is excluded from pathogenesis by our definition.

**Comment by PhiBabs935** [comment]
For the definition annotation of 'pathogenesis', we were using a short modification of the GO definition. It was in a further elaboration in an editor note that we referred to the various process parts. I suggest the following revision for the editors note for the GO class: 

"Process that is the realization of a pathogenic disposition inhering in an pathogen or pathogen population, which in the most common case has at least all of the following proper process parts: (1) pathogen transmission, (2) establishment of localization in host, (3) process of establishing an infection, and (4) appearance of disorder. 

Pathogenesis does not involve all of (1)-(4) in every case. For instance, in the case of various bacterial pathogens, pathogenesis occurs without (1)-(3) via the secretion of toxins which subsequently cause disorder in some organism."

Then we can define 'viral pathogenesis' more strictly.

**Comment by PhiBabs935** [comment]
I will remove the axiom {'preceded by' SOME 'pathogen transmission process'} from 'establishment of localization' and 'establishment of localization in host'. It seems false even for the latter. Commensal bacteria have established localization in host without a pathogen transmission process preceding. And when opportunistic pathogens infect the host this involves establishment of localization (in a new site in the host), but not a pathogen transmission process.
